test/Microsoft.Sbom.Tool.Tests/IntegrationTests.cs

```diff
@@ -307,6 +308,81 @@ public void E2E_Validate_WithValidComplianceStandard_SupportedManifestInfo_StdOu
         Assert.IsTrue(stdout.Contains("SPDXRef-Package"));
     }
 
+    [DataRow("2.2")]
+    [DataRow("3.0")]
+    [TestMethod]
+    public void E2E_Validate_WithAdditionalFile_StdOutContainsAdditionalFileErrors(string version)
+    {
+        if (!IsWindows)
+        {
+            Assert.Inconclusive("This test is not (yet) supported on non-Windows platforms.");
+            return;
+        }
+
+        var testFolderPath = CreateTestFolder();
+        GenerateManifestAndValidateSuccess(testFolderPath, manifestInfoSpdxVersion: version);
+
+        // Add additional file
+        var newFilePath = Path.Combine(testDropDirectory, "newFile");
+        File.WriteAllText(newFilePath, "This is a test file.");
+
+        var (arguments, outputFile) = GetValidateManifestArguments(testFolderPath, manifestInfoSpdxVersion: version);
+
+        var (stdout, stderr, exitCode) = LaunchAndCaptureOutput(arguments);
```

Comment:
Can we assert that the exitCode is what we expect here too, and in the other test?